exporter/splunkhecexporter/hec_worker.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package splunkhecexporter // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/splunkhecexporter"
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"net/url"
+
+	"go.opentelemetry.io/collector/consumer/consumererror"
+	"go.uber.org/multierr"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/splunk"
+)
+
+type hecWorker interface {
+	Send(context.Context, *bufferState, map[string]string) error
+}
+
+type hecWorkerWithoutAck struct {
```

Comment:
```suggestion
type defaultHecWorker struct {
```
Just to avoid setting too many expectations that ack support is coming right after, and show this is default behavior.